src/app.module.ts

```diff
@@ -0,0 +1,36 @@
+import { Module } from '@nestjs/common';
+import { AppController } from './app.controller';
```

Comment:
Please remove includes of the non-existing files.

Reply:
updated